src/ESPEasyWifi.ino

```diff
@@ -75,6 +75,10 @@
 bool WiFiConnected() {
   START_TIMER;
 
+  #ifdef HAS_ETHERNET
+  return eth_connected;
+  #endif
```

Comment:
This will give a compiler warning or maybe error when `HAS_ETHERNET` is defined.
There is some "unreachable code" after this statement.

Reply:
I will change it to
`#ifdef HAS_ETHERNET
  return eth_connected;
#else
....
  #endif`

Reply:
Change pushed to https://github.com/kretzp/ESPEasy/tree/feature/ethernet_support